ts/src/pro/bitget.ts

```diff
@@ -613,9 +614,10 @@ export default class bitget extends bitgetRest {
             }
         }
         const instId = (type === 'spot') ? marketId : 'default'; // different from other streams here the 'rest' id is required for spot markets, contract markets require default here
+        const channel = isStop ? 'ordersAlgo' : 'orders';
         const args = {
             'instType': instType,
-            'channel': 'orders',
+            'channel': channel,
```

Comment:
also it seems that stop orders updates are not correctly handled 
```
 onMessage {
  action: 'snapshot',
  arg: { instType: 'umcbl', channel: 'ordersAlgo', instId: 'default' },
  data: [
    {
      actualPx: '50.000000000',
      actualSz: '0.000000000',
      cOid: '1041586673338097665',
      cTime: '1684059535345',
      eps: 'api',
      hM: 'double_hold',
      id: '1041586673338097666',
      instId: 'LTCUSDT_UMCBL',
      key: '1041586673338097666',
      ordPx: '55.000000000',
      ordType: 'limit',
      planType: 'pl',
      posSide: 'long',
      side: 'buy',
      state: 'not_trigger',
      sz: '0.100000000',
      tS: 'open_long',
      tgtCcy: 'USDT',
      triggerPx: '55.000000000',
      triggerPxType: 'mark',
      triggerTime: '1684059535345',
      userId: '3704614084',
      version: 1041586673291960300
    }
  ]
}
```
usually when changing the subscription we also have to check `handleOrder/handleUpdate`